src/nav/navigationSelectors.js

```diff
@@ -11,3 +11,6 @@ export const getCurrentRouteParams = createSelector(
   getNavigationIndex,
   (routes, index) => routes[index].params,
 );
+
+export const getCurrentRouteName = (state: GlobalState): Object[] =>
```

Comment:
Let's just shorten it to `getCurrentRoute`. Also, the type of the returned value should be `string`.